Makefile

```diff
@@ -74,7 +82,7 @@ doc: ## compile doc as html and launch doc web server
 
 lint: ## lint the code and check coding conventions
 	@echo "Running linter..."
-	@"./node_modules/.bin/eslint" ./packages/react-admin/src/
+	@"./node_modules/.bin/eslint" ./packages/react-admin/src/ ./packages/ra-core/src/
```

Comment:
you should lint `ra-ui-materialui`, too

Reply:
In fact, we should lint all our projects ;)

Reply:
`./packages/**/src` ?